concent_api/verifier/tasks.py

```diff
@@ -179,6 +183,33 @@ def blender_verification_order(
     for file_path in source_files_list + [source_package_path]:
         delete_file(file_path)
 
+    blender_output_file_name = generate_blender_output_file_name(scene_file)
+    upload_file_name = generate_upload_file_name(subtask_id)
+    (file_size, file_check_sum) = get_file_size_and_checksum(blender_output_file_name)
```

Comment:
Let's keep consistently calling it `checksum`, not `check_sum`. Keeping it consistent makes it easier to find stuff in the code when grepping or searching on github.

Reply:
:+1: